### PR TITLE
Add support for repeating tests

### PR DESCRIPTION
**Pull request by junpataleta**
It would be great to add support for PHPUnit's --repeat argument in
order to run tests repeatedly which is useful for testing random
PHPUnit errors.

### CODE REVIEW

File: mdk/commands/phpunit.py [hunk 1]
```diff
@@ -142,13 +150,26 @@ def run(self, args):
         if testsuite and not testsuite.endswith('_testsuite'):
             testsuite += '_testsuite'
 
+        # Check repeat arg.
+        repeat = args.repeat
+        if repeat:
+            try:
+                # Make sure repeat is an integer and is greater than 1.
+                repeat = int(repeat)
```
Comment (FMCorz) [inline mdk/commands/phpunit.py]: Use `type=int` to let the parser validate the argument.

File: mdk/commands/phpunit.py [hunk 2]
```diff
@@ -103,6 +103,14 @@ class PhpunitCommand(Command):
                 'metavar': 'filter'
             }
         ),
+        (
+            ['--repeat'],
+            {
+                'default': None,
+                'help': 'run tests repeatedly',
```
Comment (FMCorz) [inline mdk/commands/phpunit.py]: If you intend for the repeat to be limited to a certain number of repeats, please clarify it in the help. Not, however, that it makes it more difficult for the user (and the parser) when a flag can have an optional flag, e.g. `--repeat`, `--repeat 20`.

I would advocate for another flag which sets the number of repeats. 

```
--repeat Repeats the test after running them
--repeat-no {int} The number of times to repeat the test
```

The second one may implicitly assume that the former is true.

File: mdk/commands/phpunit.py (hunk 1 above)
Comment (junpataleta) [reply mdk/commands/phpunit.py]: Thanks, Fred! Good tip! I have fixed this: https://github.com/FMCorz/mdk/pull/185/files#diff-3ddec6e445fe163398be1bc2f84844d4R112

File: mdk/commands/phpunit.py (hunk 2 above)
Comment (junpataleta) [reply mdk/commands/phpunit.py]: I updated the help text to be a bit more descriptive. My intention is that it should take a number, similar to PHPUnit's `--repeat` option.

**Comment by junpataleta** [comment]
Merging this as discussed with Fred. Woot!

**Pull request merged**